shopinvader_custom_attribute/__manifest__.py

```diff
@@ -6,13 +6,13 @@
 {
     "name": "Shopinvader Custom Attribute",
     "summary": "Integrate your custom attribute in your website",
-    "version": "10.0.1.0.0",
+    "version": "12.0.1.0.0",
     "category": "e-commerce",
     "website": "https://akretion.com",
     "author": "Akretion",
     "license": "AGPL-3",
-    "application": True,
-    "installable": False,
+    "application": False,
+    "installable": True,
     "external_dependencies": {"python": [], "bin": []},
     "depends": ["shopinvader", "product_custom_attribute"],
```

Comment:
this module comes from you own repo and is not migrated yet https://github.com/shopinvader/odoo-pim

Reply:
Thanks. I was starting from an early migr in odoo-ak-incubator.
In odoo-pim the pr is close to be merged https://github.com/shopinvader/odoo-pim/pull/6

Indeed the current PR is waiting for his dependency.